### ISSUE TITLE
The display ratio is not correct when running on Android, and the screen is also very blurry

### ISSUE THREAD

**Issue by leechuichui** [EdwardLu2018/wasm-ar#23]
![10071691567923_ pic](https://github.com/EdwardLu2018/wasm-ar/assets/10825690/ba9c7a72-ebb6-41e0-849e-46540c34c5c1)

**Comment by EdwardLu2018** [comment]
Yeah, same for iPhone... Mobile support needs to be updated, unfortunately. Been a while since I've updated the repo.
